src/main/scala/nl.knaw.dans.easy.bagindex/server/BagIndexServletComponent.scala

```diff
@@ -136,9 +136,8 @@ trait BagIndexServletComponent {
         .flatMap(uuid => databaseAccess.doTransaction(implicit c => database.getBagInfo(uuid)))
         .map(createResponse[BagInfo](bagInfo => <result>{toXml(bagInfo)}</result>)(bagInfo => "result" -> toJson(bagInfo)))
         .recoverWith {
-          case BagIdNotFoundException(_) => Success(createResponse[Unit](_ => <result/>)(_ => "result" -> JArray(List.empty))(()))
+          case BagIdNotFoundException(_) => Success(NotFound(createResponse[Unit](_ => <result/>)(_ => "result" -> JArray(List.empty))(())))
```

Comment:
I'd rather see a nice description telling the `bagId` doesn't exist, than an empty object in the body.